giraffe/src/components/Plot.tsx

```diff
@@ -37,10 +44,18 @@ export const Plot: FunctionComponent<PlotProps> = props => {
           {children}
         </PlotResizer>
       </div>
+    ) : (
+      <TableResizer
+        config={config as SizedConfig}
+        height={config.height}
+        width={config.width}
+      >
+        {children}
+      </TableResizer>
     )
   }
 
-  return (
+  return hasPlotEnv(config) ? (
```

Comment:
what is the relationship between `plotEnv` and `layerCanvasRef`? it seems like not having `plotEnv` means the `layerCanvasRef` isn't necessary? is that try? why or why not?

Reply:
The `layerCanvasRef` is passed directly to the layers that use it for rendering such as `lineLayer` and `rectLayer`.

The `plotEnv` deals with the state of a plot, such as its tick marks, margins, and current config which in turn affects its spec which is what different layer types use for their own state.

Reply:
Since tables do not use a canvas a `layerCanvasRef` is not necessary. `plotEnv` is also not necessary for tables because they have limited state that they handle on their own.